scripts/devnet-restart.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+HOST=127.0.0.1
+PORT=5050
+
+echo "Emptying state on Devnet at $HOST:$PORT"
+
+curl -X POST "http://$HOST:$PORT/restart"
```

Comment:
I am slightly worried about relying on the `restart` functionality instead of actually terminating and respawning devnet. But if you haven't experienced any problems with it so far...

Reply:
Nope, worked nicely for me.